src/developer.md

```diff
@@ -21,7 +21,7 @@ Please note that the following documentation is adapted from the [BestieTemplate
 If this is the first time you work with this repository, follow the instructions below to clone the repository and create a `dev` version. 
 
 !!! tip "`dev` version of a Julia package"
-    Having a `dev` (development) version of a Julia package allows you to import a local version of the package with your changes instead of the registered package version (which is static).
+    Having a `dev` (development) version of a Julia package allows you to import a local version of the package with your changes instead of the registered package version (which is static). Remeber, that could be changes in the code which are not yet registered. 
```

Comment:
```suggestion
    Having a `dev` (development) version of a Julia package allows you to import a local version of the package with your changes instead of the registered package version (which is static). Remember, there could be changes merged in the main-branch, that are not yet in any registered version. 
```